schema/artist/index.js

```diff
@@ -454,7 +482,7 @@ export const ArtistType = new GraphQLObjectType({
           },
           sort: SaleSorts,
         },
-        resolve: ({ id }, options, request, { rootValue: { relatedSalesLoader } }) =>
+        resolve: ({ id }, options, _request, { rootValue: { relatedSalesLoader } }) =>
```

Comment:
It's not apparently obvious where the `_request` is coming from. Is this some sort of convention?

Reply:
Yes, this signifies that the parameter will be unused (convention is to precede with a `_`).